src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -16,8 +16,8 @@ public class When_broadcasting_a_command : NServiceBusAcceptanceTest
         {
             var context = await Scenario.Define<Context>()
                 .WithEndpoint<Sender>(b => b.When(c => c.EndpointsStarted, (session, c) => session.Send(new Request())))
-                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.ScaleOut().InstanceDiscriminator("1")); })
-                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.ScaleOut().InstanceDiscriminator("2")); })
+                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.AddUniquelyAddressableQueue("1")); })
```

Comment:
@bording The problem is that `AddUniquelyAddressableQueue` indicates that it is a queue. So without knowing more I would wonder: Will there be a queue with the name `1`? How about `QueueDescriminator`?

But still not sure if this really aligns with 

https://github.com/Particular/PlatformDevelopment/issues/621

The name `instance` definitely did.